### ISSUE TITLE
error of Invalid Request after ERPNEXT update from 6.4 to 6.4.6

### ISSUE THREAD

**Issue by omnison** [frappe/erpnext#4118]
today i updated erpnext to 6.4.6, and when i login into erp, error information "Invalid Request" will popup, and i cannot operate. 
I power off droplet and power on again, and same issue. 

Please tell how to fix this issue.

**Comment by omnison** [comment]
<img width="970" alt="7128227f-d8f1-4a38-adc0-151542d22e48" src="https://cloud.githubusercontent.com/assets/7886153/10266503/352f277c-6a79-11e5-8f57-e3c5a8972a8a.png">

**Comment by anandpdoshi** [comment]
Close the tab, go to login page and login again. This is a security
improvement.

**Comment by omnison** [comment]
ok, let me try again.

**Comment by omnison** [comment]
now no issue in ie browser, but still problem in chrome

**Comment by anandpdoshi** [comment]
Do the same. Just close all tabs that you have and start afresh. I
understand that this is annoying but we couldn't figure out a way to make
it easier.
